consensus/src/runway/mod.rs

```diff
@@ -238,23 +293,25 @@ where
                     self.on_parents_response(u_hash, parents)
                 }
                 Response::NewestUnit(response) => {
-                    let salt = response.as_signable().salt;
-                    trace!(target: "AlephBFT-runway", "{:?} Response parents received {:?}.", self.index(), salt);
+                    trace!(target: "AlephBFT-runway", "{:?} Response parents received.", self.index());
```

Comment:
I guess it should be "Response newest unit received"? Also maybe we should print `response.index()` ?

Reply:
Better now I hope.